codalab/model/mysql_model.py

```diff
@@ -2,13 +2,44 @@
 MySQLModel is a subclass of BundleModel that stores metadata on a MySQL
 server that it connects to with the given connect parameters.
 '''
-from sqlalchemy import create_engine
+from sqlalchemy import (
+    create_engine,
+    event,
+    exc,
```

Comment:
This is actually an intentionally styling to avoid comma-related bugs when editing the imports by line (and make diffs look nicer)
http://stackoverflow.com/questions/11597901/why-does-python-allow-a-trailing-comma-in-list